src/app/components/POCs/CamelDSLCodeEditor/camelDSLutils.ts

```diff
@@ -0,0 +1,18 @@
+import { SchemasSettings, setDiagnosticsOptions } from "monaco-yaml";
+import schema from "./camelDSLSchema.json";
+export { default } from "react-monaco-editor";
+
+const defaultSchema: SchemasSettings = {
+  uri: "http://console.redhat.com/docs",
```

Comment:
This uri will end up in the popover showed for errors (See the "Invalid Source" story, move the mouse over the highlighted error).
It's not possible to hide it, unfortunately. We could put a link to the docs, when they'll be ready.